kitsune/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -444,18 +444,18 @@ <h2 class="sumo-card-heading">
   <nav class="tabs">
     <ul class="tabs--list subtopics">
       <li class="tabs--item">
-      <button class="tabs--link is-active" id="kb-tab-all" 
+      <button class="tabs--link is-active" id="kb-tab-all"
         data-event-category="device-migration-tabbed-menu"
-        data-event-action="tab-click" 
+        data-event-action="tab-click"
         data-event-label="view-all">
         <span>{{ _('View All') }}</span>
       </button>
       </li>
       {% for subtopic in subtopics %}
         <li class="tabs--item">
-          <button class="tabs--link" id="kb-subtopic-{{ subtopic.title }}" 
+          <button class="tabs--link" id="kb-subtopic-{{ subtopic.title }}"
```

Comment:
In the initial implementation of the tabbed menu, we were using the `subtopic.title` but we swapped that for the `slug` b/c of the [concern](https://github.com/mozilla/kitsune/pull/5475#discussion_r1192606494) that introducing spaces can be prone to errors. I agree that title is better especially for GA, I just want to make sure that we don't foresee any problems with the spaces.